ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -37,6 +37,7 @@
 {%                 if configure_mlag_ibgp_peering %}
         {{ switch.mlag_peer_l3_ip | arista.avd.default(switch.mlag_peer_ip) }}:
           peer_group: {{ bgp_peer_groups.MLAG_IPv4_UNDERLAY_PEER.name | arista.avd.default("MLAG-IPv4-UNDERLAY-PEER") }}
+          description: {{ switch.mlag_peer | arista.avd.default(switch.mlag_peer_l3_ip, switch.mlag_peer_ip) }}
```

Comment:
A simpler implementation would be to just set the description on the peer-group. This will inherit to all neighbors in all VRFs unless they have a description set.